src/reducers/entities/users.js

```diff
@@ -87,6 +87,32 @@ function currentUserId(state = '', action) {
     return state;
 }
 
+function myAcceptedTermsOfServiceId(state = '', action) {
+    switch (action.type) {
+    case UserTypes.RECEIVED_TERMS_OF_SERVICE_STATUS:
+        return action.data.terms_of_service_id;
+
+    case UserTypes.LOGOUT_SUCCESS:
+        return '';
+
+    default:
+        return state;
+    }
+}
+
+function myTermsOfServiceAcceptedAt(state = 0, action) {
```

Comment:
Do we really need two separate reducers here? Seems like they are both used in the same place and at the same times

Reply:
I've changed this to a single reducer here. Should I refactor this to a single selector as well?